pkg/utils/utils_test.go

```diff
@@ -396,3 +400,141 @@ func TestValidateSubModule(t *testing.T) {
 		})
 	}
 }
+
+func Test_ShuffleLocators(t *testing.T) {
+	logger, err := testutils.GetLogger()
+	if err != nil {
+		t.Errorf("Couldn't initialize logger, error: %v", err)
+	}
+	locatorArrValue := []core.LocatorConfig{{
+		Locator: "Locator_A"},
+		{
+			Locator: "Locator_B"},
+		{
+			Locator: "Locator_C"}}
+
+	type args struct {
+		locatorArr      []core.LocatorConfig
+		locatorFilePath string
+	}
+
+	tests := []struct {
+		name string
+		args args
+	}{
+		{"Test_shuffleLocators",
```

Comment:
Can you also add test for expecting error like what should happen when locators file path is incorrect or locator config is incorrect ?